copy_this/application/commands/cacheclearcommand.php

```diff
@@ -48,12 +48,20 @@ public function help(oxIOutput $oOutput)
     public function execute(oxIOutput $oOutput)
     {
         $oInput = $this->getInput();
+
+        $oOutput->writeLn('Clearing OXID db and backend cache..');
+        /** @var oxCache $oCache */
+        $oCache = oxNew('oxcache');
```

Comment:
@ddpkts It may cause confusion so users would wonder what is the difference between two of them. I personally am in favor of having single `cache:clear` command but with flag options such as `--files-only`.